pkg/operator/operator.go

```diff
@@ -39,6 +56,15 @@ type Metrics struct {
 	// objects. It is split in the dimensions of Kubernetes objects and
 	// corresponding actions (add, delete, update).
 	triggerByCounter *prometheus.CounterVec
+
+	mtx       sync.RWMutex
```

Comment:
do you mind to quickly document which fields this mutex is protecting? A simple `// protects all fields below` should do the trick.